src/index.html

```diff
@@ -38,6 +38,37 @@
     <!-- End Header -->
 
     <!-- Start Page Content -->
+    <main>
+      <section class="main-hero flex flex-col inner-content-max-width mxauto">
+        <div class="flex flex-col site-padding-x">
+          <p class="text-body color-gray pb2 sm:pb1_75 md:pb2_5">Hi, we're Seaborne.</p>
+          <h1 class="text-header color-black pb2 sm:pb1_75 md:pb2_5">
+            We're your guides to sustainable business practices.
+          </h1>
+          <a
+            aria-label="Get in touch with Seaborne"
+            class="button--style-primary text-button flex items-center justify-center"
+          >
+            Get in touch
+          </a>
+        </div>
+        <div class="hero-image">
+          <img
+            class="w100"
+            src="img/hero-illustration-md.png"
+            srcset="
+              img/hero-illustration-sm.png 240w,
+              img/hero-illustration-md.png 400w,
+              img/hero-illustration-lg.png 720w
+            "
+            sizes="(min-width: 1200px) 720w,
+            (max-width: 768px) 400w,
+            (max-width: 380px) 240w"
+            alt="Illustration of people doing outdoor activies in the mountains"
```

Comment:
"activities"! Sorry, proofreading eyeballs.

Reply:
Thank you, good catch!!